src/Data/Functor/Linear.hs

```diff
@@ -50,9 +49,17 @@ class Functor f => Applicative f where
   f <*> x = liftA2 ($) f x
   liftA2 :: (a ->. b ->. c) -> f a ->. f b ->. f c
   liftA2 f x y = f <$> x <*> y
+-- TODO: Add monad and monad fail
+-- (>>=) :: m a ->. (a ->. m b) -> m b
```

Comment:
Data Monad is not very interesting, as far as I can tell (as do notation doesn't work). Let's not rush there until we have an actual use-case.